### PR TITLE
Classification feature using regexes

### PR DESCRIPTION
**Pull request by Eazhi**
Adds the possibility to autoclassify some transaction by passing an `account_patterns` to the importer

There are also some helper functions to generate an account_pattern from existing transactions

### CODE REVIEW

File: beancount_n26/__init__.py [hunk 1]
```diff
@@ -138,6 +144,19 @@ def __init__(
 
         self._translation_strings = _translation_strings_for(self.language)
 
+        for account, payees in self.account_to_payees.items():
+            for payee in payees:
+                if payee in self.payee_to_account:
+                    logging.warning(f"{payee} in multiple accounts")
```
Comment (siddhantgoel) [inline beancount_n26/__init__.py:150]: Should we raise an error here instead?

**Review by siddhantgoel — CHANGES_REQUESTED** [review]
Nice, thanks for the PR!

Could you please add a few test cases for this feature as well?

**Comment by rougeth** [comment]
This is exactly what I was looking for. Hope this get merged soon, great job @Eazhi!

File: beancount_n26/__init__.py (hunk 1 above)
Comment (Eazhi) [reply beancount_n26/__init__.py]: Didn't really know what to raise here so I just raised an AssertionError

**Comment by Eazhi** [comment]
This should be ready to go, but I stil believe that it's better to export this classification part outside of the importer that way it can be used with any importer  
I've done this here: https://github.com/Eazhi/beancount-categorizers

**Comment by siddhantgoel** [comment]
Awesome, thanks for the PR! I'll try to push this in another release this weekend.

**Comment by siddhantgoel** [comment]
Just published v0.6.0 which includes this.

**Pull request merged**